### PR TITLE
add rc 200 and 0 to touchstone config for router-perf and modifying default number of clients

### PR DESCRIPTION
**Pull request by qiliRedHat**
1. The current touchstone config router-perf is using is https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/utils/touchstone-configs/mb-touchstone.json, it gets latency_95pctl. While avg_latency is also important, to get a feeling of the overall latency. In the real test, avg_latency is smaller than latency_95pctl. For example:
 
```
11-28 15:16:35.289      "avg_latency": 20011,
11-28 15:16:35.289      "latency_95pctl": 52734,
11-28 15:16:35.289      "latency_99pctl": 59614,
```
2. Getting the number of response code is useful to evaluate if the test load is too big for a cluster, for example when the total connection number is bigger than the cluster's capacity, more '0' response code is seen. This is useful when testing with connection related features, like https://docs.google.com/spreadsheets/d/1AvpLioEZB-HigjaKMNRa_GYiffVBiyP8arqvlm0BAZg/edit#gid=313345561.  I used to manually collect those data from output of the router-perf test. 

This PR is to add a touchstone config file to generate results for the 200 and 0 rc codes. 
I didn't include all kinds of rc code, because other rc codes are rarely seen, getting all of them can make the results sheet not easy to read.

### CODE REVIEW

**Comment by qiliRedHat** [comment]
Test result in this result sheet https://docs.google.com/spreadsheets/d/118stkJ5uzEhYCo8k9F6J0CfkEmepoE1EvCzVxmlhuxM/edit#gid=831215729

**Comment by rsevilla87** [comment]
Hey @qiliRedHat , rather than adding another config file, why don't you add the new comparisons to the original?

**Comment by qiliRedHat** [comment]
@rsevilla87 If adding to the origin don't impact others, I'm more than happy to do that.

**Comment by qiliRedHat** [comment]
@rsevilla87  Please help to review again.

File: workloads/router-perf-v2/common.sh [hunk 1]
```diff
@@ -24,13 +24,13 @@ get_scenario(){
   if [[ ${NUM_NODES} -ge ${LARGE_SCALE_THRESHOLD} ]]; then
     log "Large scale scenario detected: #workers >= ${LARGE_SCALE_THRESHOLD}"
     export NUMBER_OF_ROUTES=${LARGE_SCALE_ROUTES:-500}
-    CLIENTS=${LARGE_SCALE_CLIENTS:-"1 20 80"}
-    CLIENTS_MIX=${LARGE_SCALE_CLIENTS_MIX:-"1 10 20"}
+    CLIENTS=${LARGE_SCALE_CLIENTS:-"1 80"}
```
Comment (rsevilla87) [inline workloads/router-perf-v2/common.sh:27]: cc: @jtaleric @dry923, changing default number of clients, as per 4.12 maxconns has been upgraded from 20K to 50K. Some context -> https://github.com/cloud-bulldozer/e2e-benchmarking/issues/519#issuecomment-1331964622

**Review by rsevilla87 — APPROVED** [review]
lgtm!

**Review by jtaleric — APPROVED** [review]
lgtm

**Pull request merged**